### PR TITLE
Explicitly use "Pruébalo en CodePen"

### PR DESCRIPTION
**Pull request by alejandronanez**
When we used `[](codepen:....)` it was getting translated this to `Try this on CodePen`, making this explicit we ensure that we always get the Spanish version.

Relates to #158

### CODE REVIEW

File: content/docs/hello-world.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ El más pequeño de los ejemplos de React se ve así:
 
 ```js
 ReactDOM.render(
-  <h1>¡Hola mundo!</h1>,
+  <h1>Hello, world!</h1>,
```
Comment (alejandronanez) [inline content/docs/hello-world.md]: We missed this one!

**Comment by Darking360** [comment]
Yes, I tried as @carburo mentioned and changing the line 62 of `gatsby-config.js` we can add the default text for these parts, like this:

`defaultText: '<b>Pruébalo en CodePen</b>',`

Maybe we should add it like that 🤔 @alejandronanez

**Comment by alejandronanez** [comment]
@Darking360 @carburo I didn't know this could be done that way. I'll update my PR tonight or tomorrow if you guys want to jump in to ship this fast, feel free to do so! ✌️

File: content/docs/hello-world.md (hunk 1 above)
Comment (carburo) [reply content/docs/hello-world.md]: There is now also an edit for this in #174.

**Pull request merged**